README.md

```diff
@@ -61,7 +63,20 @@ rust_repositories()
 load("@io_bazel_rules_rust//:workspace.bzl", "bazel_version")
 bazel_version(name = "bazel_version")
 ```
-The rules are under active development, as such the lastest commit on the master branch should be used. `master` currently requires Bazel >= 0.17.0.
+The rules are under active development, as such the lastest commit on the master branch should be used. `master` currently requires Bazel >= 0.26.0.
+
+## Example
+
+Here's an example to build a Hello World:
+
+```python
+load("@io_bazel_rules_rust//rust:rust.bzl", "rust_binary")
+
+rust_binary(
+    name = "rs",
+    srcs = ["hello.rs"],
+)
+```
```

Comment:
I am doubtful this is useful. There is link for specific rules that have individual examples that are more explanatory.

Reply:
Removed.

When I tried the rule, I didn't find the label to load in the documentation. But now, I see it in the examples.